cli/flox/src/commands/environment.rs

```diff
@@ -968,7 +972,7 @@ impl List {
 
         let mut env = self
             .environment
-            .detect_concrete_environment(&flox, "list using")?
+            .detect_concrete_environment(&flox, "List using")?
```

Comment:
nonblocking
```suggestion
            .detect_concrete_environment(&flox, "List packages in which")?
```

Reply:
That doesn't really make sense for the `--config` option

Reply:
Ah true